src/main/java/filter/lexer/Lexer.java

```diff
@@ -26,7 +26,7 @@
 
             new Rule("\\(", TokenType.LBRACKET),
             new Rule("\\)", TokenType.RBRACKET),
-            new Rule("\\\"", TokenType.QUOTE),
+            new Rule(".*\\\"(.*)\\\".*", TokenType.QUOTED_CONTENT),
```

Comment:
Hmm, this is a bit too permissive. Right now things like `body:@#$%^"is"` pass tests. Try to match the smallest possible set of strings which constitute 'something that's quoted'. For example, we probably don't want to allow `"""` (which also works now).

More tests for this would also be nice.